src/views/Sale/components/SelfBidList/index.tsx

```diff
@@ -124,21 +124,93 @@ export function SelfBidList({ sale, clearingPrice, bids, isFixed }: SelfBidListP
       formatBigInt(clearingPrice.tokenOut, sale.tokenOut.decimals)
     : 0
 
-  if (isFixed) {
+  if (isFixed && isSaleClosed(sale)) {
     return (
       <Flex maxHeight="200px" flexDirection="column" style={{ position: 'relative' }}>
         <Flex flexDirection="row" alignItems="center" marginBottom="8px" padding={isMobile ? '0 8px' : '0 16px'}>
-          <Flex flex={isMobile ? 2.5 : 3}>
+          <Flex flex={isMobile ? 2.5 : 3} justifyContent="center">
             <ColumnLabel>Type</ColumnLabel>
           </Flex>
-          <Flex flex={3}>
+          <Flex flex={3} justifyContent="center">
             <ColumnLabel>Amount</ColumnLabel>
           </Flex>
-          <Flex flex={isSaleClosed(sale) ? 3 : 6} flexDirection="row" alignItems="center">
+          <Flex flex={3} flexDirection="row" justifyContent="center" alignItems="center">
+            <ColumnLabel>Value</ColumnLabel>
+          </Flex>
+          <Flex flex={isMobile ? 1 : 3} flexDirection="row" alignItems="center" justifyContent="flex-start">
+            <ColumnLabel>Status</ColumnLabel>
+          </Flex>
+        </Flex>
+        <Flex style={{ overflowY: 'scroll' }} flexDirection="column">
+          {bids.map((bid: FixedPriceSalePurchase, index: number) => {
+            const bidValue =
+              formatBigInt(sale.tokenPrice, sale.tokenOut.decimals) * formatBigInt(bid.amount, sale.tokenOut.decimals)
+
+            return (
+              <Flex
+                key={index}
+                flexDirection="row"
+                alignItems="center"
+                minHeight="50px"
+                borderTop="1px dashed #DDDDE3"
+                padding={isMobile ? '0 8px' : '0 16px'}
+              >
```

Comment:
I also think something like this works better being defined as a separate styled component, makes it more readable

Reply:
Since there are a lot of them above a style.tsx might be a good addition